docs/development.md

```diff
@@ -48,6 +48,8 @@ To gather the probe data that populates the probe API, run the following:
 ./manage.py import_probes
 ```
 
+The next step requires viewer permissions in the non-prod GCP project, please reach out to someone on the #glam Slack channel if you need the proper authorization. Then, log in to GCP with [`gcloud auth application-default login`](https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login) to be able to pull data using terminal commands.
```

Comment:
I think prettier might want this line wrapped?

Also instead of "pull data" maybe "import data"?